docker/scripts/remove-backup-archives

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
+
+set -e -o pipefail -u
+
+DUMP_DIR="${MUSICBRAINZ_SEARCH_DUMP_DIR:-/media/searchdump}"
+
+SCRIPT_NAME=$(basename "$0")
+HELP=$(cat <<EOH
+Usage: $SCRIPT_NAME [<options>]
+
+Remove MusicBrainz Solr collection backup archives.
+
+Options:
+  -h, --help    Print this help message
+
+Environment:
+  MUSICBRAINZ_SEARCH_DUMP_DIR   path to directory with Solr backup archives
+                                (default: /media/searchdump)
+EOH
+)
+
+# Parse arguments
+
+if [[ $# -gt 0 && $1 =~ ^-*h(elp)?$ ]]
+then
+	echo "$HELP"
+	exit 0 # EX_OK
+elif [[ $# -gt 0 ]]
+then
+	echo >&2 "$SCRIPT_NAME: unrecognized arguments"
+	echo >&2 "Try '$SCRIPT_NAME help' for usage."
+	exit 64 # EX_USAGE
+fi
+
+# Just remove MusicBrainz Solr collection backup archives/dumps
+
+dump_size="$(du -h "$DUMP_DIR" | cut -f1))"
+rm -frv "${DUMP_DIR:?}/*"
```

Comment:
Would it be possible to use a more specific pattern for the dump files to avoid accidental file removal, in case `DUMP_DIR` is incorrectly specified?